trafilatura/utils.py

```diff
@@ -322,12 +322,14 @@ def normalize_authors(current_authors, author_string):
         author = AUTHOR_REMOVE_NUMBERS.sub('', author)
         author = AUTHOR_REMOVE_PREPOSITION.sub('', author)
         # skip empty strings
-        if len(author) == 0:
+        if len(author) == 0 or (' ' not in author and len(author) >= 23):
```

Comment:
Is there a good reason to set 23 as threshold? What about other languages than English?

Reply:
The average of the first name is 6 characters, for english and portuguese.  I believe maybe German first name have a few more letters.  To set a number I used this ISO https://en.m.wikipedia.org/wiki/ISO/IEC_7813

Reply:
Thanks for the reference! It says 26 characters including separators, but there are also composite names with a hyphen. It would be a pity to miss information by being too restrictive here.

Reply:
Constraint relaxed in 8eb2605.